locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,178 @@
+dashboard-exposures-area-description-premium = We found your information exposed { $exposures_unresolved_num } times over { $data_breach_unresolved_num } data breaches and { $data_broker_unresolved_num } data broker sites that are selling your personal info.
+
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+dashboard-exposures-breaches-scan-progress-description = {
+  $exposures_unresolved_num ->
+    [one] {
+      $data_breach_unresolved_num ->
+        [one] We found your information exposed { $exposures_unresolved_num } time in { $data_breach_unresolved_num } data breach. We’re still scanning sites that may be selling your personal info.
+        *[other] We found your information exposed { $exposures_unresolved_num } time in { $data_breach_unresolved_num } data breaches. We’re still scanning sites that may be selling your personal info.
+    }
+    *[other] {
+      $data_breach_unresolved_num ->
+        [one] We found your information exposed { $exposures_unresolved_num } times in { $data_breach_unresolved_num } data breach. We’re still scanning sites that may be selling your personal info.
+        *[other] We found your information exposed { $exposures_unresolved_num } times in { $data_breach_unresolved_num } data breaches. We’re still scanning sites that may be selling your personal info.
+    }
+}
+dashboard-exposures-no-breaches-scan-progress-description = We didn’t find any data breaches, but we’re still scanning sites that may be selling your personal info.
+dashboard-exposures-scan-progress-label = Scan in progress
+
+dashboard-fixed-area-headline-premium = View all exposures that are fixed or in-progress
+
+# Variables:
+# $data_broker_total_num is the total number of data brokers selling the user’s data.
+dashboard-exposures-all-fixed-free-scan = {
+    $data_broker_total_num ->
+      [one] Next <a>start your free scan</a> of { $data_broker_total_num } site that may be selling your personal info.
+     *[other] Next <a>start your free scan</a> of { $data_broker_total_num } sites that may be selling your personal info.
+  }
+
+## False door test
+
+# Strings used in a banner (false door test) to observe engagement with Monitor premium and gauge user interest.
+
+false-door-test-content-part-one = No one should be able to buy your personal information.
+false-door-test-content-part-two = Automatically remove data from sites trying to sell it.
+false-door-test-content-part-two-dashboard = Auto-delete data from sites trying to sell it.
+false-door-test-cta = Count me in
+false-door-test-popup-close = Close
+false-door-test-phase-2-content-part-one = Did you know there are sites selling your private info?
+false-door-test-phase-2-content-part-two = Automatically remove data from sketchy sites.
+false-door-test-phase-2-content-part-two-dashboard = Automatically remove data from broker sites. 
```

Comment:
```suggestion
false-door-test-phase-2-content-part-two-dashboard = Automatically remove data from broker sites.
```

Just noticed this one has a weird non-breaking space at the end.

Reply:
Oh huh, how did that get there? 2677cf78f